linux/device.go

```diff
@@ -1234,3 +1315,37 @@ func ExpandDevice(targetPath string, volAccessType model.VolumeAccessType) error
 	}
 	return err
 }
+
+func resizeMappedLuksDevice(devPath string) (bool, error){
+	log.Debugf(">>>> resizeMappedLuksDevice - %s", devPath)
+	defer log.Debugf("<<<< resizeMappedLuksDevice - %s", devPath)
+
+	_, exitStatus, _ := util.ExecCommandOutput("cryptsetup", []string{"resize", devPath})
+	log.Debugf("exit status of resize LUKS device: %d", exitStatus)
+	if exitStatus == 0 {
+		log.Infof("mapped LUKS device %s resized successfully", devPath)
+		return true, nil
+	}
+	msg := fmt.Errorf("failed to resize mapped LUKS device %s", devPath)
+	log.Debugln(msg.Error())
+		return false, msg
+}
+
+func isMappedLuksDevice(devPath string) (bool, error) {
+	log.Debugf(">>>> isMappedLuksDevice - %s", devPath)
+	defer log.Debugf("<<<< isMappedLuksDevice - %s", devPath)
+
+	_, exitStatus, _ := util.ExecCommandOutput("cryptsetup", []string{"status", "-v", devPath})
```

Comment:
ExecCommandOutput returns stdout and stderr in a single string, the return code, and error.
Can we log some more information from the command output ?

Reply:
sure